src/hotspot/share/utilities/debug.cpp

```diff
@@ -278,21 +286,18 @@ void report_java_out_of_memory(const char* message) {
 
 class Command : public StackObj {
  private:
-  ResourceMark rm;
-  bool debug_save;
+  ResourceMark _rm;
+  DebuggingContext _debugging;
```

Comment:
Why a different initialization syntax here?

Reply:
I don't understand the question?  These are member variable declarations.

Reply:
I believe David is asking about why these particular member names were changed

Reply:
No I'm comparing the syntax used elsewhere:
```
DebuggingContext debugging{};
```
with that used here:
```
DebuggingContext debugging;
```
why the braces in most cases but not here?

Reply:
"syntax used elsewhere" are declarations of local variables initialized with _value initialization_ (which is _default initialization_ for this type).  "that used here" is a member variable declaration in the Command class, where the initialization occurs elsewhere (in the constructor of the Command class [*]).  They are entirely different syntactic entities.  Does that help?  Or am I still not understanding your question?

[*] With C++11, data members can have default initializers, but that's not an approved feature for HotSpot.

Reply:
Sorry I was fixated on the weirdness (to me) of the {} initialization. I missed the fact these were different kinds of declarations.